velox/exec/MergeSource.cpp

```diff
@@ -181,8 +181,8 @@ class MergeExchangeSource : public MergeSource {
   }
 
  private:
-  MergeExchange* mergeExchange_;
-  std::unique_ptr<ExchangeClient> client_;
+  MergeExchange* const mergeExchange_;
+  std::shared_ptr<ExchangeClient> client_;
```

Comment:
Why is this a shared_ptr?

Reply:
No need. Changed it to unique_ptr. Thanks!